SOS/Finance/Broker.cpp

```diff
@@ -11,6 +11,8 @@ using namespace std;
 
 namespace finance
 {
+//	const string data_table_filename = "C:\\Temp\\PushP\\push - 3.1.0\\test - Search\\dataTable.csv";
+	const string data_table_filename = "C:\\dataTable.csv";
```

Comment:
Restore to get data file from proper location.

Reply:
Change "C:\\dataTable.csv" to "C:\\Temp\\PushP\\push - 3.1.0\\test - Search\\dataTable.csv".